dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/GithubAppCheck.java

```diff
@@ -97,18 +103,24 @@ protected boolean isGithubAppEnabledOnRepository(String fullRepoName) {
      */
     protected boolean isGithubAppEnabledOnRepositoryWithGitApi(String fullRepoName) {
         refreshJwtIfNeeded(appId, privateKeyPath);
-        try {
-            gitHub.getApp().getInstallationByRepository(fullRepoName.split("/")[0], fullRepoName.split("/")[1]);
-            return true;
-        } catch (HttpException exception) {
-            if (exception.getResponseCode() != 404) {
-                // Log for any HTTP status code other than 404 Not found. 
-                log.warn("Caught a HTTPException {} while trying to get app installation. Defaulting to False", exception.getMessage());
+        try (CloseableHttpClient httpClient = HttpClients.createDefault()) {
```

Comment:
Clients are heavy weight objects.  Usually you create one for the class and reuse it for all requests.

Reply:
@cjneasbi Done